### PR TITLE
Navigation bar fixes

### PR DESCRIPTION
**Pull request by Adda0**
This PR fixes navigation bar sharp edges around buttons caused by incorrect grouping of buttons.

### CODE REVIEW

File: ClientApp/KachnaOnline/src/app/home/repeating-states/repeating-states.component.html [hunk 1]
```diff
@@ -3,11 +3,11 @@ <h2>Pravidelná otevírací doba</h2>
 </div>
 
 <div class="d-flex justify-content-center border-bottom pb-2 mb-2">
-  <h5 *ngIf="hasRegularOpenings == undefined" class="text-muted">
+  <h5 *ngIf="hasRegularOpenings == undefined" class="text-muted text-center ">
```
Comment (ondryaso) [inline ClientApp/KachnaOnline/src/app/home/repeating-states/repeating-states.component.html:6]: Why? The text is centered by its parent flexbox

File: ClientApp/KachnaOnline/src/app/home/repeating-states/repeating-states.component.html [hunk 2]
```diff
@@ -3,11 +3,11 @@ <h2>Pravidelná otevírací doba</h2>
...
     Zjišťuji pravidelnou otevírací dobu...
   </h5>
 
-  <h5 *ngIf="hasRegularOpenings != undefined && !hasRegularOpenings">
+  <h5 *ngIf="hasRegularOpenings != undefined && !hasRegularOpenings" class="text-center">
```
Comment (ondryaso) [inline ClientApp/KachnaOnline/src/app/home/repeating-states/repeating-states.component.html:10]: Same

**Review by ondryaso — CHANGES_REQUESTED** [review]

File: ClientApp/KachnaOnline/src/app/home/repeating-states/repeating-states.component.html (hunk 1 above)
Comment (Adda0) [reply ClientApp/KachnaOnline/src/app/home/repeating-states/repeating-states.component.html:6]: No, it is aligned to the left for the hardcoded messages.

File: ClientApp/KachnaOnline/src/app/home/repeating-states/repeating-states.component.html (hunk 2 above)
Comment (Adda0) [reply ClientApp/KachnaOnline/src/app/home/repeating-states/repeating-states.component.html:10]: No, it is aligned to the left for the hardcoded messages.

**Pull request merged**